.github/workflows/tars-ci.yml

```diff
@@ -48,6 +48,7 @@ jobs:
 
       - name: Linux Install
         run: |
+          sudo apt-get update
           sudo apt install -y cpanminus build-essential libncurses5-dev libreadline-dev libssl-dev perl libpcre3 libpcre3-dev libldap2-dev
```

Comment:
Would be better to put repeated commands in https://github.com/apache/apisix/blob/master/ci/common.sh?

Reply:
update